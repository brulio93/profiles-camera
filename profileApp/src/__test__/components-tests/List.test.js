import React from 'react';
import Enzyme from 'enzyme';
import MyList from '../components/list/List';
import renderer from 'react-test-renderer';
const ProfileStore = require('../stores/ProfileStore').default;

describe('ProfilesList', function() {
    it("render filtered profiles", function(){
        const render = renderer.create(<MyList ProfileStore={ProfileStore.ProfileStore} />);
        expect(render).toMatchSnapshot();
    });
});
