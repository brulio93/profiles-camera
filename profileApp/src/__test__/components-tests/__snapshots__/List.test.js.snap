// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilesList render the component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <inject-MyList-with-ProfileStore
    ProfileStore={ProfileListStore {}}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "class",
    "props": Object {
      "ProfileStore": ProfileListStore {},
    },
    "ref": [Function],
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "ProfileStore": ProfileListStore {},
      },
      "ref": [Function],
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
